sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCV2Suite.scala

```diff
@@ -2396,7 +2396,7 @@ class JDBCV2Suite extends QueryTest with SharedSparkSession with ExplainSuiteHel
         |SELECT
         |  REGR_AVGX(bonus, bonus),
         |  REGR_AVGY(bonus, bonus)
-        |FROM h2.test.employee WHERE dept > 0 GROUP BY DePt""".stripMargin)
+        |FROM h2.test.employee WHERE dept > 0 GROUP BY DEPT""".stripMargin)
```

Comment:
This is just test. Let's just leave it. Also in this way it can test different case sensitivity

Reply:
Thank you for your explanation and review❤️. I feel that the necessity of this PR is not significant.